apps/src/applab/applab.js

```diff
@@ -1747,9 +1747,10 @@ Applab.imageUploadButton = function (opts) {
 };
 
 // These offset are used to ensure that the turtle image is centered over
-// its x,y coordinates. The image is currently 31x45, so these offsets are 50%
-var TURTLE_X_OFFSET = -15;
-var TURTLE_Y_OFFSET = -22;
+// its x,y coordinates. The image is currently 48x48, rendered at 24x24.
+var TURTLE_X_OFFSET = -12;
+var TURTLE_Y_OFFSET = -12;
+var TURTLE_ROTATION_OFFSET = -45;
```

Comment:
Would it make more sense to just rotate the image that we check in by 45 degrees instead of having this offset in code?

I might also try to make the x/y offset a little less mysterious, either setting them to "-24 / 2" (a little more clear why this number) or by creating a separate TURTLE_WIDTH/TURTLE_HEIGHT and setting to -TURTLE_WIDTH / 2